src/par/src/TPPartitioner.cpp

```diff
@@ -380,34 +367,109 @@ void TPpartitioner::OptimalPartCplexWarmStart(
                              e_wt_factors_.begin(),
                              0.0);
     unvisited_hyperedges.insert(std::pair<int, float>(e, score));
+    tot_cost += score;
   }
-
-  int max_num_hyperedges = 0;
-  std::vector<int> edge_mask;
-  float base_score = (*unvisited_hyperedges.begin()).second / 10.0;
+  // set the reserved total score for reduced hypergraph (100% for now)
+  const float reserved_tot_cost = tot_cost * 1.0;
```

Comment:
No need to * 1.0 but won't this lead to all edges being included and none pruned?

Reply:
Leave 1.0 there.  We want to tune this parameter to better runtime and reasonable runtime later.